ext/date/date_core.c

```diff
@@ -7796,7 +7796,7 @@ datetime_s_now(int argc, VALUE *argv, VALUE klass)
 	s = 59;
 #ifdef HAVE_STRUCT_TM_TM_GMTOFF
     of = tm.tm_gmtoff;
-#elif defined(HAVE_TIMEZONE)
+#elif defined(HAVE_TIMEZONE) && !(defined __MINGW32__ && defined _UCRT)
```

Comment:
I think we no longer need this change.
See ruby/ruby@2bd6c5dc1650e852a95961402b094ccf0909842b.

Reply:
https://github.com/ruby/ruby/commit/2bd6c5dc1650e852a95961402b094ccf0909842b works. May I rebase this PR to master and remove this commit?

Reply:
Yes, please.
At least rebasing and resolving a conflict is needed.

Reply:
It's rebased.